src/Workspaces/Core/Portable/Workspace/Host/HostProjectServices.cs

```diff
@@ -6,16 +6,18 @@
 
 namespace Microsoft.CodeAnalysis.Host
 {
-    // TODO(cyrusn): Make public.  Tracked through https://github.com/dotnet/roslyn/issues/62914
-    internal sealed class HostProjectServices
+    /// <summary>
+    /// Per language services provided by the host environment.
+    /// </summary>
+    public sealed class LanguageServices
     {
         private readonly HostLanguageServices _services;
 
-        public HostSolutionServices SolutionServices => _services.WorkspaceServices.SolutionServices;
+        public SolutionServices SolutionServices => _services.WorkspaceServices.SolutionServices;
 
         // This ensures a single instance of this type associated with each HostLanguageServices.
         [Obsolete("Do not call directly.  Use HostLanguageServices.ProjectServices to acquire an instance")]
-        internal HostProjectServices(HostLanguageServices services)
+        internal LanguageServices(HostLanguageServices services)
         {
             _services = services;
         }
```

Comment:
note: GetRequiredService below could become an extension method.  But that has the downside that the caller must know to include a using for `Microsoft.CodeAnalysis.Host` to see that extension.  so it's just convenient to have thse be instance methods instead.

Reply:
Instance methods are definitely better. Extension methods are not discoverable.